candi/control-plane-kubeadm/config.yaml.tpl

```diff
@@ -169,7 +169,7 @@ controllerManager:
     node-monitor-period: "{{ .arguments.nodeMonitorPeriod }}s"
     node-monitor-grace-period: "{{ .arguments.nodeMonitorGracePeriod }}s"
   {{- end }}
-  {{- if and (hasKey .arguments "podEvictionTimeout") (semverCompare "< 1.27" .clusterConfiguration.kubernetesVersion) }}
+  {{- if and (hasKey .arguments "podEvictionTimeout") (semverCompare "= 1.27" .clusterConfiguration.kubernetesVersion) }}
```

Comment:
there's a mistake here, pod-eviction-timeout is [removed](https://github.com/kubernetes/kubernetes/pull/115840) in version 1.27